BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
```

Comment:
Consider using Dependency Injection to instantiate "ExchangeRateService" instead of creating it directly. This allows better management of dependencies and makes the service easier to test.